examples/offline/fetch_her_ddpg.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import os
+import pprint
+from typing import Any, Dict, Optional, Sequence, Tuple, Union
+
+import gym
+import numpy as np
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.data import Collector, HERReplayBuffer, HERVectorReplayBuffer
+from tianshou.data.batch import Batch
+from tianshou.env import ShmemVectorEnv, TruncatedAsTerminated
+from tianshou.exploration import GaussianNoise
+from tianshou.policy import DDPGPolicy
+from tianshou.trainer import offpolicy_trainer
+from tianshou.utils import TensorboardLogger, WandbLogger
+from tianshou.utils.net.common import Net
+from tianshou.utils.net.continuous import Actor, Critic
+
+
+def get_args():
+    # python3 fetch_her_ddpg.py --task FetchReach-v3 --seed 0 --horizon 50
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--task", type=str, default="FetchReach-v3")
```

Comment:
Is it possible to apply HER to other tasks, e.g., HalfCheetah-v3, Ant-v3, so that we can compare performances? If not, can you run DDPG on FetchReach-v3 with `mujoco_ddpg.py` to get a baseline number? Then you can update `examples/mujoco/README.md` with both numbers, i.e., DDPG with and without HER.